pkg/osquery/tables/execparsers/remotectl/parse_test.go

```diff
@@ -0,0 +1,113 @@
+//go:build darwin
+// +build darwin
+
+package remotectl
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParse(t *testing.T) {
+	t.Parallel()
+
+	var tests = []struct {
+		name        string
+		input       []byte
+		expectedErr bool
+	}{
+		{
+			name:        "empty input",
+			input:       []byte("\n"),
+			expectedErr: false,
+		},
+		{
+			name:        "dumpstate with single device in output",
+			input:       readTestFile(t, path.Join("test-data", "single_device_dumpstate.txt")),
```

Comment:
You can avoid the ReadFile boilerplate and replace these with references to package vars defined something like:

```
//go:embed test-data/single_device_dumpstate.txt
var single_device_dumpstate string
```

as an example: https://github.com/kolide/launcher/blob/main/pkg/osquery/tables/apple_silicon_security_policy/table_test.go#L54

Reply:
Oh cool, thank you!! Will do